examples/phi3/phi3_vision.py

```diff
@@ -0,0 +1,362 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import argparse
+import glob
+import json
+import os
+import readline
+from pathlib import Path
+
+from olive.common.utils import run_subprocess
+from olive.workflows import run as olive_run
+from olive.workflows.run.config import RunConfig
+
+# flake8: noqa: T201
+# phi3-vision only supports CPU and CUDA targets for now
+TARGETS = ["cpu", "cuda"]
+config_path = Path(__file__).parent / "vision" / "config_templates"
+
+
+def get_args(raw_args):
+    parser = argparse.ArgumentParser(description="phi3 optimization")
+
+    parser.add_argument(
+        "--target",
+        type=str,
+        default="cpu",
+        required=False,
+        choices=TARGETS,
+        help=f"Choose from {TARGETS}",
+    )
+    parser.add_argument(
+        "--precision",
+        type=str,
+        default="int4",
+        choices=["int4"],
```

Comment:
float models are not supported?

Reply:
Add fp16 support. The fp32 of onnx model run failed, 

[E:onnxruntime:onnxruntime-genai, sequential_executor.cc:516 ExecuteKernel] Non-zero status code returned while running Expand node. Name:'/model/attn_mask_reformat/input_ids_subgraph/Expand' Status Message: /model/attn_mask_reformat/input_ids_subgraph/Expand: left operand cannot broadcast on dim 3 LeftShape: {1,1,2500,2500}, RightShape: {1,1,2500,5572}